### PR TITLE
V1.1.0

### PR DESCRIPTION
**Pull request by glmxndr**

### CODE REVIEW

File: .scr/_init.sh [hunk 1]
```diff
@@ -142,11 +142,11 @@ prod)
   # Tag to be used by image builds, and where to deploy the images
   VERSION=${VERSION:-${ENVTYPE}-${S2USER}}
   DEST_REGISTRY=${DEST_REGISTRY:-$PROD_REGISTRY_URL}
-  BASE_REGISTRY=${BASE_REGISTRY:-$PROD_REGISTRY_URL}
```
Comment (ilaborie) [inline .scr/_init.sh:145]: Are you sure ?

File: .scr/_init.sh [hunk 2]
```diff
@@ -142,11 +142,11 @@ prod)
...
+  BASE_REGISTRY=${BASE_REGISTRY:-$IVV_REGISTRY_URL}
   INTERNAL_REGISTRY=${INTERNAL_REGISTRY:-$PROD_REGISTRY_INTERNAL_URL}
 
   # Where to find kafka client and server
-  KAFKA_URL=${KAFKA_PROD_URL:-"prd-cce-csgroup-kafka-client.prd-cce-csgroup-kafka.esa-csc-s2-prd-cce-orchestrator.svc.cluster.local:9092"}
```
Comment (ilaborie) [inline .scr/_init.sh:149]: Are you sure ?

**Review by ilaborie — CHANGES_REQUESTED** [review]
IMHO `.src/_init.sh` should be rollbacked

File: .scr/_init.sh (hunk 1 above)
Comment (glmxndr) [reply .scr/_init.sh:145]: Yes. there are no base images in the prod registry. Base images are only in the IVV registry.

File: .scr/_init.sh (hunk 2 above)
Comment (glmxndr) [reply .scr/_init.sh:149]: Yes. This host does not work, and kafkacat is smart enough to find the correct host when needed.

**Review by ilaborie — APPROVED** [review]

**Pull request merged**